examples/deploy/README.md

```diff
@@ -0,0 +1,14 @@
+### Example production deployment of bokeh server.
+
+Execute each of the following in separate shells
+
+1.  `redis-server --port 7001`
+2.  `gunicorn -k tornado -w 4 "bokeh.server.start:make_tornado(config_file='config.py')" --log-level=debug --log-file=- -b 0.0.0.0:5006`
```

Comment:
Couple of thoughts:

* Can `log-level` and `log-file` be set inside `config.py`?
* Where does `config.py` come from?  If it's shipped with the code and is always there, why not make it the default `config_file` value?